addons/onion-skinning/userscript.js

```diff
@@ -584,6 +584,10 @@ export default async function ({ addon, console, msg }) {
   settingButton.title = msg("settings");
   settingButton.appendChild(createButtonImage("settings"));
 
+  document.body.addEventListener("click", (e) => {
+    if (!e.target.matches("[class*=sa-onion-]") && areSettingsOpen()) setSettingsOpen(false);
```

Comment:
Nitpick, but can you check if the settings are open first?

Reply:
Yeah I guess we shouldn't bother checking any elements if the settings arent open, doing that now